### ISSUE TITLE
Port twistd script to py3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8012]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#8012|
|Type|enhancement|
|Created|2015-08-31 12:20:14Z|
|Branch|https://github.com/twisted/twisted/tree/twistd-py3-8012|

The hard work was already done to port trial but we should also enable the twistd script and have a line in the release notes.

porting should include bin/twistd and scripts/twistd and scripts/_twistd_unix

<details><summary>Searchable metadata</summary>

```
trac-id__8012 8012
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__branches_twistd_py3_8012 branches/twistd-py3-8012
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1441023614155486 1441023614155486
changetime__1441162328833259 1441162328833259
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [45599]) Branching to twistd-py3-8012.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl removed owner|
|-|-|

Builders look good, please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Thanks. All good. Please merge!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45606]) Merge twistd-py3-8012: Port twistd to Python 3

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8012](https://github.com/twisted/twisted/issues/8012)
